### ISSUE TITLE
Keras: Support for continuously adapting neural nets with production data ?

### ISSUE THREAD

**Issue by anujgupta82** [fchollet/keras#1095]
I have created a simple neural net (using keras) for my NLP application which classify data point into 1 of the 2 classes {Class labels: 0,1}. My model gave > 95% accuracy on Test data. I have deployed the model in production. Its running good. The prediction output of the model is sent as an email to users.

These end users can send back a feedback - if the prediction was right or not. I want to incorporate this feedback into the model so as to continuously learn from this feedback (both positive and negative feedback) and improve

How can I achieve the above ? I have read about Reinforcement Learning, Online Learning, Active Learning and Adaptive learning. Any pointers/tutorials (possibly with code) will be of great help.

**Comment by EderSantana** [comment]
"Feedback in neural nets" is a term already taken by recurrent neural networks and such. I'd recommend you changing your title to continuously adapting neural nets with production data or something.

Beyond the ideas you saw in active learning, the naive way I would propose is waiting for data enough to form a batch and do an update step with your model.

If I find out other ideas I'll let you know.
